lib/compiler-singlepass/src/codegen_x64.rs

```diff
@@ -1754,6 +1781,14 @@ impl<'a> FuncGen<'a> {
 
         // TODO: Full preemption by explicit signal checking
 
+        // We insert set StackOverflow as the default trap that can happen
+        // anywhere in the function prologue for sake of simplicity.
```

Comment:
```suggestion
        // anywhere in the function prologue.
```
StackOverflow covering the function prologue is correct.